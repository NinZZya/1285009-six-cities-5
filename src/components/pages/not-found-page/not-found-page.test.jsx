import React from 'react';
import {Router} from 'react-router-dom';
import renderer from 'react-test-renderer';
import NotFoundPage from './not-found-page';
import history from '../../../constants/history';


describe(`Should NotFoundPage render correctly`, () => {
  it(`Should NotFoundPage render correctly`, () => {
    const tree = renderer
      .create(
          <Router history={history}>
            <NotFoundPage />
          </Router>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
