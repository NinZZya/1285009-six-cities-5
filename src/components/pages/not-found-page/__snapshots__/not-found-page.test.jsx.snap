// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should NotFoundPage render correctly Should NotFoundPage render correctly 1`] = `
<main
  className="page__main page__main--index page__main--index-empty"
>
  <div
    className="cities"
  >
    <div
      className="container cities__places-container cities__places-container--empty"
    >
      <section
        className="cities__no-places"
      >
        <div
          className="cities__status-wrapper tabs__content"
        >
          <b
            className="cities__status"
          >
            Page not found
          </b>
          <p>
            <a
              className="button form__submit"
              href="/"
              onClick={[Function]}
            >
              Click here to visit the main page
            </a>
          </p>
        </div>
      </section>
      <div
        className="cities__right-section"
      />
    </div>
  </div>
</main>
`;
